catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -293,28 +298,81 @@ void registryInfoQuery() throws UnsupportedQueryException {
         SourceResponse identityMetacard = query(queryRequest);
         if (identityMetacard.getResults()
                 .size() > 0) {
-            remoteName = identityMetacard.getResults()
+            String metacardTitle = identityMetacard.getResults()
                     .get(0)
                     .getMetacard()
                     .getTitle();
+            setId(identityMetacard.getResults()
+                    .get(0)
+                    .getMetacard()
+                    .getTitle());
             registryId = RegistryUtility.getRegistryId(identityMetacard.getResults()
                     .get(0)
                     .getMetacard());
+            updateConfiguration(metacardTitle);
         }
-        updateConfiguration();
+
     }
 
-    private void updateConfiguration() {
+    private void updateConfiguration(String metacardTitle) {
+        if (metacardTitle == null) {
+            LOGGER.error("Unable to update registry configurations. No metacard title.");
+            return;
+        }
         String currentPid = getConfigurationPid();
         try {
             Configuration currentConfig = configAdmin.getConfiguration(currentPid);
             Dictionary<String, Object> currentProperties = currentConfig.getProperties();
-            currentProperties.put(REMOTE_NAME, remoteName);
-            currentProperties.put(RegistryObjectMetacardType.REGISTRY_ID, registryId);
-            currentConfig.update(currentProperties);
+            URI uri;
+            String registryUrl = "";
+            if (currentProperties.get("registryUrl") != null) {
+                registryUrl = (String) currentProperties.get("registryUrl");
+            } else if (currentProperties.get("url") != null) {
+                LOGGER.warn("current properties had url");
```

Comment:
Maybe more of a debug level message? Also could provide more info on what is happening.